src/main/java/org/spongepowered/common/mixin/core/world/MixinExplosion.java

```diff
@@ -56,11 +56,21 @@
     @SuppressWarnings("rawtypes")
     @Shadow public List affectedBlockPositions;
 
+    private boolean shouldBreakBlocks;
```

Comment:
It is, in `ExplosionBuilder`, `shouldBreakBlocks` is passed in in place of `isSmoking` and `isSmoking` is set separately; I could change this or comment it if you think this is a problem.